README.md

```diff
@@ -343,7 +343,15 @@ public void Connect()
 Send mail to the Mailtrap server for testing purposes. Uses the MailKit SMTP implementation.
 ```csharp
 public MailtrapClient(string username, string password)
-    : base(GetSettings(username, password))
+    : base(GetSettings(username, password, HostType.Test))
+{
+}
+```
+
+You can also use Mailtrap sever for sending production emails.

```

Comment:
Could you add an example of creating a `MailtrapClient` instance here, to show that passing in `HostType.Production` is the way to achieve this?

Reply:
added an example